apps/web/app/page.tsx

```diff
@@ -3,12 +3,13 @@
 import { JSX } from "react";
 // repo
 import Link from "@repo/ui/link";
+import { URL_WALKTHROUGH_HREF } from "@repo/constants/src/urls";
 
 export default function Page(): JSX.Element {
   return (
     <div className="container">
       <p>
-        <Link href="/walkthrough/9.9.9/">Walkthrough 9.9.9</Link>
+        <Link href={`${URL_WALKTHROUGH_HREF}/9.9.9/`}>Walkthrough 9.9.9</Link>
```

Comment:
Do we think this string will be used in more than one place? should we create a constant here?

Reply:
Which part of the string are you referring to? Are you referring to the additional 9.9.9 part?

Reply:
Yes, the 9.9.9 part? I'm okay either way

Reply:
Possibly, but I also see this part being refactored when we actually build this landing page and the cards on it. I'd rather wait until later to see if we need to introduce a const until then. Because honestly the one that's there is only used once now too and probably not needed at this point.